File: screenshot.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({ headless: true });
  const page = await browser.newPage({ viewport: { width: 1280, height: 800 } });
  
  await page.goto('http://localhost:8080', { waitUntil: 'networkidle' });
  await page.screenshot({ path: '/tmp/site-screenshot.png', fullPage: true });
  
  console.log('Screenshot saved to /tmp/site-screenshot.png');
  await browser.close();
})();

File: screenshot.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page(viewport={"width": 1280, "height": 800})
    page.goto("http://localhost:8080", wait_until="networkidle")
    page.screenshot(path="/tmp/site-screenshot.png", full_page=True)
    print("Screenshot saved to /tmp/site-screenshot.png")
    browser.close()
